core/embed/io/ble/inc/io/ble.h

```diff
@@ -170,6 +188,9 @@ bool ble_write(const uint8_t *data, uint16_t len);
 // Check if read is possible
 bool ble_can_read(void);
 
+// Get bond list
+uint8_t ble_get_bond_list(bt_le_addr_t *bonds, size_t count);
+
```

Comment:
If it is not currently available - would it be possible also to return the currently connected device address?
IIUC, it is currently stored at `ble_driver_t`:
https://github.com/trezor/trezor-firmware/blob/29465d65bf13d81690737b303f4cd3702834a228/core/embed/io/ble/stm32/ble.c#L54-L55

It would allow us to mark the connected device as such:
https://www.figma.com/design/Dkl7W5PLqpr3TnXJ5hbnfd/--Future-Prod---Safe-7?node-id=12331-41763&t=5e0OU6ASG2SPXG08-4

Reply:
> Or maybe we could add it to state returned by the ble_get_state function.

Sounds good - thanks!